requirements.txt

```
@@ -1 +1 @@
-xmlschema[codegen]==1.11.3
+xmlschema[codegen]==3.0.2
```

```suggestion
xmlschema[codegen]~=3.0.2
```
This would enable a micro update, but only to compatible versions (not 3.x with x>0, probably). Tested with Python 3.8.

`~=` is added in PEP-508 and defined in PEP-440, but I cannot figure out [which Python (or PIP) version includes those](https://peps.python.org/pep-0000/). 😕 But should be fine for all our versions, since we require 3.8 anyway.

But, that said, xmlschema 3.1.0 was released a few hours ago. 😎

---

Good point. I have adjusted it. But I would leave it at 3.0.2 for now :sweat_smile: